ansible/roles/openshift_outbound_tcp_logging/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+- set_fact:
+    outbound_logging_chain:
+    - chain: oo-outbound-logging
+      line:  ":oo-outbound-logging - [0:0]"
+      after: "filter$"
+      regex: "Create oo-outbound-logging chain"
+      test: True
+      pos: 1
+
+    outbound_logging_rules:
+    - chain: oo-outbound-logging
+      line:  "-A oo-outbound-logging -m comment --comment \"Log outbound TCP connections with this chain\" -m limit --limit 50/second --limit-burst 50 -j LOG --log-prefix \"tcp_out_logging: \""
+      after: "^-A FORWARD"
+      regex: "Log outbound TCP connections with this chain"
+      test: True
+      pos: 1
+
+    - chain: oo-outbound-logging
+      line:  "-A oo-outbound-logging -m comment --comment \"We use this pattern since there is no LOG_AND_RETURN target\" -m limit --limit 50/second --limit-burst 50 -j RETURN"
+      after: "Log outbound TCP connections with this chain"
+      regex: "We use this pattern since there is no LOG_AND_RETURN target"
+      test: True
+      pos: 2
+
+    - chain: oo-outbound-logging
+      line: "-A oo-outbound-logging -m comment --comment \"Drop 1 or more logs for 60 seconds after this message\" -m limit --limit 1/minute --limit-burst 1 -j LOG --log-prefix \"tcp_out_logging: logdrop \"" 
```

Comment:
Same thing goes here w.r.t. comments. This comment doesn't need to be in the iptables rule, but should be document this chain here in this YAML file.